src/math/curve-evaluator.js

```diff
@@ -1,7 +1,20 @@
 import { CURVE_CARDINAL, CURVE_CATMULL, CURVE_LINEAR, CURVE_SMOOTHSTEP, CURVE_SPLINE, CURVE_STEP } from './constants.js';
 import { math } from './math.js';
 
+/** @typedef {import('./curve.js').Curve} Curve */
+
+/**
+ * A class for evaluating a curve at a specific time.
+ *
+ * @ignore
+ */
 class CurveEvaluator {
+    /**
+     * Create a new CurveEvaluator instance.
+     *
+     * @param {Curve} curve - The curve to evaluate.
+     * @param {number} time - The initial time to evaluate the curve at. Defualts to 0.
```

Comment:
*looked up the Irish meaning* 😅 
```suggestion
     * @param {number} time - The initial time to evaluate the curve at. Defaults to 0.
```

Reply:
Well spotted - thanks! 🙏